pkgs/by-name/qu/quickemu/package.nix

```
@@ -40,7 +40,7 @@ let
       pciutils
       procps
       python3
-      qemu
+      (qemu.override { smbdSupport = true; })
```

Should we consider the host platform as `qemu_full` does?

```suggestion
      (qemu.override { smbdSupport = lib.meta.availableOn stdenv.hostPlatform samba; })
```

---

No, because evidently quickemu fails to run when QEMU is built without Samba support?

---

I'm not sure if that is a good idea, since you seem to need smbd support for quickemu to work, so it wouldn't make sense on platforms where samba is not available?

---

I think it depends on the VM image one uses with `quickemu`, probably not all require `samba`, but it's an extensive list to go through one by one